cpp/llm_chat.cc

```diff
@@ -709,6 +801,19 @@ class LLMChat {
     return Downcast<NDArray>(ret[0]);
   }
 
+  // run forward compute with embeddings
```

Comment:
Can we move it to relax side? Or at least make it llama specific because the index are hardcoded for relax.

Reply:
pending for Ruihang's param manager pr in relax